app/controllers/reservations_controller.rb

```diff
@@ -16,6 +16,8 @@ class ReservationsController < ApplicationController
   include ReservationSwitch
   helper TimelineHelper
 
+  MAX_RESERVATIONS_PERIOD = 40.days
```

Comment:
Was this number already used somewhere?

Reply:
Nope, but if it the index endpoint is only used for calendars it should be enough